src/game/Objects/Creature.cpp

```diff
@@ -215,7 +215,7 @@ Creature::Creature(CreatureSubtype subtype) :
 
 Creature::~Creature()
 {
-    CleanupsBeforeDelete();
+    Unit::CleanupsBeforeDelete();
```

Comment:
why explicitly calling unit version

Reply:
Just for readability, it's done the same way at Player's destructor. Both Creature and Player use the Unit version